lambdas/evaluator/handler.py

```diff
@@ -0,0 +1,241 @@
+import json
+import requests
```

Comment:
let's group imports by built-in.
# built in
import json
import base64
import os
from os.path ...

import requests
import dotenv ...
from openai ...